github-actions/start-release/start_release.py

```diff
@@ -69,6 +67,9 @@ def build_release_notes(release_version, unreleased_notes, app_json):
                              .format(note))
         release_notes.append(match.group())
 
+    if len(release_notes) == 2:
+        return None
```

Comment:
haha - I was on the fence about commenting on this magic number ... I generally try to avoid that kind of feedback, but yeah, in this case, it probably makes sense to change it since the significance of 2 isn't obvious

Reply:
good point, I've updated the code to check against a flag